.taprc

```diff
@@ -0,0 +1,3 @@
+# vim: set filetype=yaml :
+timeout: 60
+plugin: []
```

Comment:
`.taprc` is updated each time tests are run.
It adds the vim modeline and the set plugin to `[]`.
When validating `git commit -a -m`, considering the githook precommit that runs the tests, this is the final result.
I guess it's more to to keep it that way?